dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/HttpUtilsTest.java

```diff
@@ -40,10 +40,10 @@ public class HttpUtilsTest {
     @Test
     public void testGetTest() {
 	// success
-	String result = HttpUtils.get("https://github.com/manifest.json");
+	String result = HttpUtils.get("https://www.baidu.com/sugrec?prod=pc_his&from=pc_web&json=1&sid=36427_36454_31253_36422_36165_36487_36055_36376_36234_26350_36469_36316&hisdata=&_t=1653904731156&req=2&csor=0");
```

Comment:
I don't think this is a good idea. Due to the instability of Baidu. This url may cause unstable in unit test. 

And If GitHub cannot be accessed, the DS code can not be pulled.

Reply:
Perhaps because of the github anti-ddos attack, different policies are used for different URLs, but multiple tries always work.

Reply:
> Perhaps because of the github anti-ddos attack, different policies are used for different URLs, but multiple tries always work.

Maybe your network is in poor condition.

Reply:
You can consider changing to some high-performance ```Airports```

Reply:
Actually I'd just remove this test case or set up an http server before the tests. Unit tests should not rely on external third party services. No matter what links you use there must be someone with poor connections and will timeout.

Reply:
> Actually I'd just remove this test case or set up an http server before the tests. Unit tests should not rely on external third party services. No matter what links you use there must be someone with poor connections and will timeout.
So do I need to merge this pr now, or do I hide these two test cases in this pr because they will cause the test to be unstable

Reply:
> > Actually I'd just remove this test case or set up an http server before the tests. Unit tests should not rely on external third party services. No matter what links you use there must be someone with poor connections and will timeout.
> 
> 
> 
> So do you need to merge this pr now, or do I hide these two test cases in this pr because they will cause the test to be unstable

Can you just modify the test to set up an http server with Jetty, and use localhost:port to test this http util? That would be stable and no timeout.